### DISCUSSION TITLE
Quick modification of code in Schedule.class.js

### DISCUSSION THREAD

**Discussion by oliviermollier** [jhuckaby/Cronicle#783] [Q&A]
Hello, i did a quick modification in the text of Schedule.class.js. I stop and restart and my modification is not visible. I notice one file called _combo.js that seems to be a concatenation of several JS.
I updated it as well but still not visible. i clean cache of course of my browser but still not ok.

any idea?

Thx

**Comment by jhuckaby** [comment]
```sh
sudo /opt/cronicle/bin/build.js dist
```
